### ISSUE TITLE
Toolbar items are not displayed correctly in DataGrip

### ISSUE THREAD

**Issue by svenbit** [thvardhan/Gradianto#13]
### What/Why

The table data editor, query console, and results table toolbar doesn't seems to be displayed correctly when using Gradianto themes in **DataGrip**.

### Steps to reproduce

1. Change appearance theme to Gradianto (Dark Fuchsia / Deep Ocean / Midnight Blue)
2. Open table data editor/query console
3. Hover on any of the item in the toolbar
4. The toolbar items displayed in the toolbar will be disoriented as you hover through the items

### Additonal Information
DataGrip: 2020.1.2 Build #DB-201.6668.164
Operating System: Windows 10 18363.778

I have also tried to reproduce the issue on another DataGrip instance installed in another PC, and I was able to reproduce the issue.

Table Data Editor:
![2020-04-26_15h33_06](https://user-images.githubusercontent.com/1471939/80302319-822c1e00-87d3-11ea-8395-5884271ab450.png)

Query Console:
![2020-04-26_15h34_19](https://user-images.githubusercontent.com/1471939/80302323-85bfa500-87d3-11ea-9d67-25a67af8054b.png)

Query Results Table:
![2020-04-26_15h34_30](https://user-images.githubusercontent.com/1471939/80302325-89ebc280-87d3-11ea-8399-5bb9f84702ec.png)

**Comment by thvardhan** [comment]
Hi thanks for reporting this issue. I'll take a look at it as soon as I get access to a computer.

P.S. really appreciate the detailed report.

**Comment by thvardhan** [comment]
Hi, I've tested the above but I am unable to reproduce the issue. Can you try updating data grid to latest version and check? it seems more like a data grid issue than theme because I am only changing the color values they should not overlap. if not then please let me know I'll check once again and if its still not fixed, then we can open an issue ticket with JetBrains.
<img width="368" alt="Screenshot 2020-05-01 at 4 30 24 PM" src="https://user-images.githubusercontent.com/17314986/80801170-301b3c00-8bc9-11ea-9e62-10f23d15108c.png">

<img width="380" alt="Screenshot 2020-05-01 at 4 29 12 PM" src="https://user-images.githubusercontent.com/17314986/80801166-2c87b500-8bc9-11ea-8fbb-c02a5b9be47c.png">

**Comment by svenbit** [comment]
Thank you for taking the time to check the issue. 

The DataGrip version that I used was already in the latest version when I submitted the report (2020.1.2 Build #DB-201.6668.164).

But after I saw the screenshot that you have attached, I realized that the toolbar background color is different compared to the toolbar color in my DataGrip instance (the toolbar background color in my DataGrip instance was light green as shown in the screenshot in my first post).

So I did further check, and it turns out that I have the [*Database Color Settings*](https://www.jetbrains.com/help/datagrip/database-color-settings-dialog.html) enabled, and I can confirm that the issue only happens if you enable the Color Settings for the database and the *Enable database colors in toolbars is checked*.

![2020-05-02_00h11_33](https://user-images.githubusercontent.com/1471939/80824949-9e7fee80-8c09-11ea-9607-b89949a553eb.png)

![2020-05-02_00h12_45](https://user-images.githubusercontent.com/1471939/80824977-af306480-8c09-11ea-97ea-1704d2b611c7.png)

I'm not sure if it is something that can be fixed from the theme configuration or it was due to a bug in DataGrip. But when I changed the appearance to other themes, the toolbar is displayed properly with the database color settings enabled.

I tried the following themes and all of them has not issue with the database color settings:
- Dark Purple (From Marketplace, by Jetbrains)
- Dracula (3rd Party)
- High Contrast (Built-in)
- Hiberbee Dark (3rd Party)
- Windows 10 Light (Built-in)

**Comment by thvardhan** [comment]
Hi I've checked for the same on a Mac as I don't have access to windows pc right now. I'll check it on windows as soon as I get access to it. using the same build on Mac seems to be fine. again as mentioned will check with windows as soon as possible.
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/17314986/80827685-d5520680-8c01-11ea-946d-4d9fb67f98bf.png">

**Comment by thvardhan** [comment]
Hi can you confirm if the bug is still there? I know it has been a while but I've just got a friends windows pc I am unable to reproduce the issue.

**Comment by svenbit** [comment]
Yes, the bug is still there for me, but it only happens if the database *Color Settings* is enabled as mentioned before.
The issue does not happen if the database color settings is not enabled (No Color).

Gradianto version: 4.2
DataGrip version: 2020.2.2 Build DB-202.6948.53

**Comment by thvardhan** [comment]
ok thanks for the headsup I'll recheck and get back

**Comment by ernstmarco** [comment]
Having same bug in Intellij while using colors for datasources:

![image](https://user-images.githubusercontent.com/5818157/114879521-91034a80-9e01-11eb-93ec-6d7d8a0aaf40.png)

Nice theme btw. 👍

**Comment by thvardhan** [comment]
Hello, it's been a while, I did a test back then but wasn't able to find anything and hence added the hacktoberfest label hoping for support, will keep this issue open for anyone coming in future.

can anyone confirm if this still happens in new UI?

**Comment by AndreiGorevoi** [comment]
> Hello, it's been a while, I did a test back then but wasn't able to find anything and hence added the hacktoberfest label hoping for support, will keep this issue open for anyone coming in future.
> 
> can anyone confirm if this still happens in new UI?

Hello @thvardhan. I confirm that the issue it still exists for the NEW UI. There are my steps for reproducing that issue:
1. Selecting any Gradianto theme;
2. Open build in tool for databases in IDEA;
![image](https://github.com/thvardhan/Gradianto/assets/56274189/b1aeecee-436a-4d9f-aad3-241c4983bd23)

3. Create  some DB connection and **select some custom color for that(this is the key of reproducing the problem)**
![image](https://github.com/thvardhan/Gradianto/assets/56274189/a86176b0-4297-454d-86a2-feba0b8941e9)
![image](https://github.com/thvardhan/Gradianto/assets/56274189/b14dce2a-73bc-435b-8136-5a4e9cce1d41)

4. Then when you work with some of these connection(open console, some table and hover over panel) with custom colour it's some weird behaviour with colours:
![image](https://github.com/thvardhan/Gradianto/assets/56274189/8c369419-decb-4d9b-9204-72372d63b5c2)

P.S. If you use DB connection/source with default colour it works fine.
